### PR TITLE
Show more debug logs when failed to build resource model due to `getIndex` fails

### PR DESCRIPTION
**Pull request by wangxf1987**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
when the "getIndex" fails, display more data details in logs.

**Which issue(s) this PR fixes**:
Fixes # add more details in the log.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by RainbowMango — COMMENTED** [review]
/assign
Thanks Xiaofei

File: pkg/modeling/modeling.go [hunk 1]
```diff
@@ -163,7 +163,7 @@ func (rs *ResourceSummary) clusterResourceNodeComparator(a, b interface{}) int {
 func (rs *ResourceSummary) AddToResourceSummary(crn ClusterResourceNode) {
 	index := rs.getIndex(crn)
 	if index == -1 {
-		klog.Error("ClusterResource can not add to resource summary: index is invalid.")
+		klog.Errorf("ClusterResource can not add to resource summary: index %d is invalid. the ClusterResourceNode is %v.", index, crn)
```
Comment (RainbowMango) [inline pkg/modeling/modeling.go:166]:
```suggestion
		klog.Errorf("Failed to add node to resource summary due to no appropriate grade. ClusterResourceNode:%v.", crn)
```

**Comment by RainbowMango** [comment]
/retitle Show more debug logs when failed to build resource model due to `getIndex` fails

**Comment by RainbowMango** [comment]
It's not certain that here is the bug, it makes sense to have more detail debug logs.
/remove-kind bug
/kind cleanup

**Comment by wangxf1987** [comment]
Finished to fix and squash it

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve
Thanks.

**Comment by yanfeng1992** [comment]
Okay, we can try to do this together. Can you contribute your grafana configuration json? @wangxf1987

**Pull request merged**